crates/proof-of-sql/src/sql/proof_plans/projection_exec_test.rs

```diff
@@ -174,7 +174,7 @@ fn we_can_get_an_empty_result_from_a_basic_projection_on_an_empty_table_using_fi
             ColumnType::Decimal75(Precision::new(75).unwrap(), 0),
         ),
     ];
-    let first_round_builder = &mut FirstRoundBuilder::new();
+    let first_round_builder = &mut FirstRoundBuilder::new(0);
```

Comment:
Same here

Reply:
these tables are empty so the lengths should be zero right